### ISSUE TITLE
Changes to Glycan classifiction

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#1787]
_This task is for the preparation of 2.8 and will not be part of the 2.7 release._

This ticket is related to #1672 and concerns the change requests for the glycan classification for @edwardsnj. The attached file summarizes the instances that should be changed.

[Glycosylation glycan error filter - Nathan.xlsx](https://github.com/user-attachments/files/17103232/Glycosylation.glycan.error.filter.-.Nathan.xlsx)

Some of the changes involes species specific exeptions:
* N-glycans with no branching in fly
* Yeast specific O-man structures

It is not clear how to deal with them. One solution could be to create separate subclasses in the classification hierarchy for them (see #1786). 

Questions can be discussed Wednesdays or via email with @ReneRanzinger.

**Comment by ReneRanzinger** [comment]
Do a first pass of the list and see on which structures you would like to push back. Mike, Rene can meet with you in a separate meeting to go over them. Once this is decided we will give them to @katewarner (#1672) to report to dataset owners. Implementation will be done after this.

**Comment by katewarner** [comment]
@edwardsnj can this ticket be closed?

**Comment by edwardsnj** [comment]
Yes, we can close this, I did a bunch of work to improve the glycan type assignment. If additional specific issues are identified, these can be made into singleton issues.
